v3/plugins/scalameta/metals/1.7.2/meta.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+publisher: scalameta
+name: metals
+version: 1.7.2
+type: VS Code extension
+displayName: Scala (Metals)
+title: Scala language server with rich IDE features
+description: This plug-in provides support for Scala development via the Metals language server.
+icon: https://rawcdn.githack.com/OlegIlyenko/scala-icon/c453c6f67dfcc1dbf0ca1e437d32d04f1d710983/scala-icon.svg
+repository: https://github.com/scalameta/metals-vscode
+category: Language
+firstPublicationDate: "2020-01-24"
+spec:
+  containers:
+    - image: "quay.io/eclipse/che-sidecar-java:11"
+      name: vscode-metals
+      memoryLimit: "1500Mi"
+      volumes:
+      - name: sbt
+        mountPath: /home/theia/.sbt
+      - name: ivy2
+        mountPath: /home/theia/.ivy2
+      - name: coursier
+        mountPath: /home/theia/.cache/coursier
```

Comment:
This volume mount is causing issues for us due to how subpaths are created in OpenShift. Since `/home/theia/.cache` does not exist, it is created when mounting the `coursier` folder. However, it is created with `755` permissions with root owner and group, and so is not writable by any process in the container.

Reply:
I shortened `/home/theia/.cache/coursier` to `/home/theia/.cache` and things seem to be working

Reply:
Yeah, that should fix the startup issue. The problem, however, is that now we're persisting the entire user `.cache` directory, which could potentially cause other problems. I think a better solution is to add volumes for both `.cache/metals` and `.cache/coursier` if those caches need to be persisted.

What are these caches used for? If we don't have them persistent, how negatively is performance impacted?

Reply:
`.cache/coursier` not being persisted means that you would have to download all packages again, every time